Formula/aws-cdk.rb

```diff
@@ -0,0 +1,24 @@
+require "language/node"
+
+class AwsCdk < Formula
+  desc "AWS Cloud Development Kit - framework for defining AWS infra as code"
+  homepage "https://github.com/aws/aws-cdk"
+  url "https://registry.npmjs.org/aws-cdk/-/aws-cdk-1.7.0.tgz"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    version = shell_output("#{bin}/cdk --version")
+    mkdir "testapp"
+    cd testpath/"testapp"
+    shell_output("#{bin}/cdk init app --language=javascript")
+    list = shell_output("#{bin}/cdk list")
+    assert_match "1.7.0 (build 8870695)", version
```

Comment:
This will break on every automated update. If you want to do a version check, please check for `#{version}`

Reply:
Done